src/components/Common/ModalOptions/OptionsElements.jsx

```diff
@@ -49,3 +41,21 @@ export const OptionsClose = styled.button`
     top: 5px;
     right: 5px;
 `;
+export const OptionsOverlay = styled.div`
+    position: fixed;
+    inset: 0;
+    background-color: rgb(0 0 0 / 20%);
+    z-index: 100;
+    cursor: default;
+`;
+export const OptionsModalContainer = styled.div`
+    width: 200px;
+    height: 160px;
+    position: relative;
+    background-color: #0e0e0e;
+    border-radius: 10px;
+    border: 1px #3d3d3d solid;
+    box-shadow: 2px 2px 10px #333;
+    padding: 20px 0;
+    z-index: 1000;
+`;
```

Comment:
Is it possible to use smaller values for the z-index? 
I was just thinking that this can cause `z-index : 10000` in the future and its not so readable

Reply:
`z-index : 10000`
Yes, what do you say about `z-index : 100`?